### ISSUE TITLE
[Feature Request]: search with replace

### ISSUE THREAD

**Issue by helmut72** [gristlabs/grist-core#159]
If I don't have overseen anything, there is a search, but no replace feature.

Thank you.

**Comment by paulfitz** [comment]
Yes, there is no replace feature, sorry about that. Some kinds of replacements can be done with formulas, or by filtering for values of interest and pasting over them - but there's no general purpose “search-and-replace” feature yet.

**Comment by helmut72** [comment]
Thanks for the hint. Yes, need to start to play with formulas.
